CMakeLists.txt

```diff
@@ -55,16 +55,6 @@ execute_process(
 )
 set(VERSION_MD5 "0x${VERSION_MD5}")
 
-if (NOT DEFINED PPFW_FILENAME)
```

Comment:
In this case you can also remove setting the filename in line 49, and just directly set it in the firmware CMakeLists without checking for existence, we just need to use the same filename in the github workflow too! (if you insist on keep using both oci and ppfw we can also change the workflow to look for that. Or even get rid of ppfw complately.